docs/proposals/27-project-group.md

```diff
@@ -0,0 +1,197 @@
+
+To summarize, there are two scenarios which need to be systematically enabled:
+
+1. Enable the assured scheduling of shoot control planes to specific geographical regions
+2. Let users have the opportunity to have a "private seed" (enabled by Gardener operators)
+
+Both scenarios are currently already technically possible, but require strong involvement of Gardener operators.
+
+Also users can easily ignore those restrictions, by simply setting the `.spec.seedSelector` of a shoot to match a seed without the taint / outside the required geographical location.
+
+In order to enforce something like this, a new resource called [`SeedBinding`s](#seedbinding) will be introduced.
+
+Furthermore, [`ProjectGroup`s](#projectgroup) should decrease the work of Gardener operators by letting users group different projects into a single project group, where seed bindings will automatically be shared among them.
+
+### Goals
+
+The overarching goal of this GEP is to lay the foundational work to enable homogeneous sovereign cloud scenarios.
+
+- Enable projects to use some seeds exclusively by using the Taints and Tolerations feature ([`SeedBinding`s](#seedbinding))
+- Enhance projects so that shoot control planes can be restricted to certain geographical regions ([`SeedBinding`s](#seedbinding))
+- Share those, so called, seed bindings to different projects that logically belong together in a concept called [`ProjectGroup`s](#projectgroup)
+- Introduce new role for project group administrators, that are allowed to add projects to a project group
+
+### Non-Goals
+
+- Let users exclusively bind seeds on their own
+
+## Proposal
+
+#### ProjectGroup
+
+A ProjectGroup API will be introduced that abstracts certain information over one or more projects. With this, it will become possible to homogeneously share resources between projects.
+
+Such a ProjectGroup manifest looks like this:
+```yaml
+apiVersion: core.gardener.cloud/v1beta1
+kind: ProjectGroup
+metadata:
+  name: group1
+spec:
+  namespace: gardenprojectgroup-group1
+  owner:
+    apiGroup: rbac.authorization.k8s.io
+    kind: User
+    name: john.doe@example.com
+  members:
+  - apiGroup: rbac.authorization.k8s.io
+    kind: User
+    name: alice.doe@example.com
+    role: admin
+  projects:
+    - Project1
+    - Project2
+```
+
+The main idea behind `ProjectGroup`s is that they, like projects, have their own namespace.
+Every resource that is shareable (at first only `SeedBinding`s) in this namespace, will be shared to all projects in this project group.
+
+This sharing mechanism will be achieved by the `ProjectGroup` controller.
+It will copy the objects from the project group namespace to the namespaces of the projects that belong to the project group.
+
+When the object in the project group namespace is changed, the controller will immediately update the objects in the namespaces of the projects.
```

Comment:
Should the controller also prevent changes in the target namespace, e.g. a project member changing the copied `SeedBinding` resource? Should this all be implemented in the controller or should we use `ManagedResource`s?

Reply:
Yes, the controller should prevent users from changing the object in the target namespace.
I am unsure how we could leverage `ManagedResource`s for this. Could you elaborate on this?